docs/consume-packages/configuring-nuget-behavior.md

```diff
@@ -58,7 +58,10 @@ Settings are managed using the NuGet CLI [config command](../reference/cli-refer
 Windows:
 
 ```cli
-# Set repositoryPath in the user-level config file
+# Set globalPackagesFolder in the user-level config file
+nuget config -set globalPackagesFolder=c:\packages
+
+# Set repositoryPath (available for packages.config only) in the user-level config file
```

Comment:
So, I see that the nuget.exe command example already exists in the previous docs, but the comment especially saying to set this at the user level config should be considered a very poor practise. If someone wants to set the repositorypath config, they should only do it in the repo nuget.config.

I didn't test it myself, but I suspect that setting repositoryPath in the user nuget.config will cause the "solution" packages folder to be in the user's nuget config directory, and therefore any packages installed will cause project file modifications to reference files under the user's home directory and therefore will break everyone if commit into source control.

Reply:
I'm considering this out of scope as I'm not going to decide whether we remove explanations in our docs in this PR.